.pre-commit-config.yaml

```diff
@@ -36,6 +36,7 @@ repos:
 - repo: local
   hooks:
     - id: check-no-tests-are-ignored
+      additional_dependencies: [pandas,pyyaml]
```

Comment:
no objection here, but pandas is a pretty heavy dependency for a pre-commit hook. I'll have a look at this later, perhaps it can be done without it

Reply:
It certainly can be done with a bunch of dictionaries, but then the summarization doesn't work with one-liners and we'd have to do our own table printing. Not sure if that's worth the effort?